src/js/view/controls/components/timeslider.js

```diff
@@ -173,11 +175,8 @@ class TimeSlider extends Slider {
     }
 
     onPlaylistItem(model, playlistItem) {
-        if (!playlistItem) {
-            return;
-        }
         this.reset();
-        model.set('cues', []);
+        this.updateCues(model, model.get('cues'));
```

Comment:
Where are cues being reset now?

Reply:
Thank you! It was working fine when I was because `cues` are being updated when they load.